styles/ide-haskell-repl.less

```diff
@@ -45,7 +63,7 @@ div.ide-haskell-repl {
     overflow-y: scroll;
     pre {
       font-size: inherit;
-      font-family: inherit;
+      font-family: monospace;
```

Comment:
Okay, I guess this means you don't have a font specified in editor config, and I didn't think of this particular case... v0.4.9 should behave as I described.